libcxx/include/tuple

```diff
@@ -939,10 +963,11 @@ public:
     }
 
     template <class _Tuple,
-              class = typename enable_if
+              typename enable_if
                       <
-                         __tuple_assignable<_Tuple, tuple>::value
-                      >::type
+                         __tuple_assignable<_Tuple, tuple>::value,
+                         void*
+                      >::type = nullptr
```

Comment:
A clever trick here is to not add the second argument, but add a `*` after `::type`. But this is fine too ;)

Reply:
I'll update this to an actual `_EnableIf` from the internal meta functions.